src/index.js

```diff
@@ -1 +1,100 @@
-import './style.scss';
\ No newline at end of file
+import './style.scss';
+
+const date = document.querySelector('.date');
+const time = document.querySelector('.time');
+const hum = document.querySelector('.humidity');
+const pres = document.querySelector('.pressure');
+const wind = document.querySelector('.wind');
+const temp = document.querySelector('.temprature');
+const degreebtn = document.querySelectorAll('.buts > button');
+const cloud = document.querySelector('.clouds');
+const btn = document.querySelector('button');
+const input = document.querySelector('input');
+const mini_Temp = document.querySelector('.mintemp');
+const maxi_Temp = document.querySelector('.maxtemp');
+const display = document.querySelector('#weather');
+const h = document.querySelector('h1');
+const weatherContainer = document.querySelector('.weather');
+const city = document.querySelector('h2');
+const today = new Date();
+
+let weatherData;
+async function weatherInfo(){
+   let data = await(await fetch(`https://api.openweathermap.org/data/2.5/weather?q=${input.value},forecast?id=524901&APPID=674a2431b0b48b1ff9777f74e58d3b58`)).json();
+    weatherData = data
+    resultdata(data);
+    return data;
+}
+
+btn.addEventListener('click', async f => {
+    f.preventDefault();
+    await weatherInfo();
+    DateandTime();
+    input.value = '';
+    
+});
+
+degreebtn.forEach(btn =>{
+    btn.addEventListener('click', f =>{
+        f.preventDefault();
+        if(btn.textContent === '°F'){
+            f.currentTarget.classList.toggle('active');
+            degreebtn[0].classList.remove('active');
+            resultdata(weatherData, 'Farenheit');
+        }else if (btn.textContent === '°C'){
+            f.currentTarget.classList.toggle('active');
+            degreebtn[1].classList.remove('active');
+            resultdata(weatherData, 'celsius');
+        }
+
+    });
+});
+
+function DateandTime(){
+    date.textContent = `${thisDate().month} ${today.getDate()}, ${today.getFullYear()} - ${
+        thisDate().day
+    }`;
+    time.textContent = `${today.getHours()}:${today.getUTCMinutes()}`;
+}
+
+function resultdata(data, degreeTemp = 'farenheit'){
+
+    const celsius = Math.round(data.main.temp - 273.2);
+    const Farenheit = Math.round(celsius * (9/5) + 32);
+
+    const maxCelsius = Math.round(data.main.temp_max - 273.2);
+    const maxFarenheit = Math.round(celsius * (9/5) + 32);
+
+    const minCelsius = Math.round(data.main.temp_min - 273.2);
+    const minFarenheit = Math.round(celsius * (9/5) + 32);
+
+  // Console.log(data);
```

Comment:
Consider deleting commented out codes. If you need to review a previous version of your code, use git history.